doc/man7/migration_guide.pod

```diff
@@ -0,0 +1,1814 @@
+OSSL_STORE_LOADER_set_open, OSSL_STORE_LOADER_set_open_ex,
+OSSL_STORE_register_loader, OSSL_STORE_unregister_loader and OSSL_STORE_vctrl
+
+These functions helped applications and engines to create loaders for
+schemes they supported.  These are all deprecated and discouraged in favour of
+provider implementations, see L<provider-storemgmt(7)>.
+
+=item PEM_read_DHparams, PEM_read_bio_DHparams, 
+
+=item PEM_read_DSAparams PEM_read_bio_DSAparams
+
+=item PEM_read_DSAPrivateKey, PEM_read_DSA_PUBKEY, PEM_read_bio_DSAPrivateKey and PEM_read_bio_DSA_PUBKEY
+
+=item PEM_read_ECPKParameters, PEM_read_ECPrivateKey, PEM_read_EC_PUBKEY,
+PEM_read_bio_ECPKParameters, PEM_read_bio_ECPrivateKey and PEM_read_bio_EC_PUBKEY
+
+=item PEM_read_RSAPrivateKey, PEM_read_RSA_PUBKEY, PEM_read_RSAPublicKey,
+PEM_read_bio_RSAPrivateKey, PEM_read_bio_RSA_PUBKEY and PEM_read_bio_RSAPublicKey
+
+=item PEM_write_bio_DHparams, PEM_write_bio_DHxparams, PEM_write_DHparams and PEM_write_DHxparams
+
+=item PEM_write_DSAparams, PEM_write_DSAPrivateKey, PEM_write_DSA_PUBKEY,
+PEM_write_bio_DSAparams, PEM_write_bio_DSAPrivateKey and PEM_write_bio_DSA_PUBKEY
+
+=item PEM_write_ECPKParameters, PEM_write_ECPrivateKey, PEM_write_EC_PUBKEY
+PEM_write_bio_ECPKParameters, PEM_write_bio_ECPrivateKey and PEM_write_bio_EC_PUBKEY
+
+=item PEM_write_RSAPrivateKey, PEM_write_RSA_PUBKEY, PEM_write_RSAPublicKey,
+PEM_write_bio_RSAPrivateKey, PEM_write_bio_RSA_PUBKEY and PEM_write_bio_RSAPublicKey.
+
+See L</Deprecated low-level key reading and writing functions>
+
+=item PKCS1_MGF1
+
+See L</Deprecated low-level encryption functions>.
+
+=item RAND_get_rand_method, RAND_set_rand_method, RAND_OpenSSL and RAND_set_rand_engine
+
+Applications should instead use L<RAND_set_DRBG_type(3)>,
+L<EVP_RAND(3)> and L<EVP_RAND(7)>.
+See L<RAND_set_rand_method(3)> for more details.
+
+=item RC2_encrypt, RC2_decrypt, RC2_set_key, RC2_cbc_encrypt, RC2_cfb64_encrypt
+RC2_ecb_encrypt and RC2_ofb64_encrypt
+
+=item RC4, RC4_set_key, RC4_options
+
+=item RC5_32_encrypt, RC5_32_set_key, RC5_32_decrypt, RC5_32_cbc_encrypt,
+RC5_32_cfb64_encrypt, RC5_32_ecb_encrypt and RC5_32_ofb64_encrypt
+
+See L</Deprecated low-level encryption functions>.
+The Algorithms "RC2", "RC4" and "RC5" have been moved to the L<Legacy Provider|/Legacy Algorithms>.
+
+=item RIPEMD160, RIPEMD160_Init, RIPEMD160_Update, RIPEMD160_Final and RIPEMD160_Transform
+
+See L</Deprecated low-level digest functions>.
+The RIPE algorithm has been moved to the L<Legacy Provider|/Legacy Algorithms>.
+
+=item RSA_bits, RSA_security_bits and RSA_size
+
+Use L<EVP_PKEY_bits(3)>, L<EVP_PKEY_security_bits(3)> and L<EVP_PKEY_size(3)>.
+
+=item RSA_check_key and RSA_check_key_ex
+
+See L</Deprecated low-level validation functions>
+
+=item RSA_clear_flags, RSA_flags, RSA_set_flags, RSA_test_flags,
+
+=item RSA_setup_blinding, RSA_blinding_off and RSA_blinding_on
+
+All of these RSA flags have been deprecated without replacement:
```

Comment:
Is _these_ referring to the two =items above of the list after the colon?
Some clarification would be nice.

> All of these functions are deprecated without replacement.  The following RSA flags have been deprecated without replacement:

Reply:
Collapsed into one..